bigbluebutton-client/locale/en_US/bbbResources.properties

```diff
@@ -358,6 +358,7 @@ bbb.logout.rejected = The connection to the server has been rejected
 bbb.logout.invalidapp = The red5 app does not exist
 bbb.logout.unknown = Your client has lost connection with the server
 bbb.logout.usercommand = You have logged out of the conference
+bbb.logout.ejectedFromMeeting = You have been ejected from the meeting.
```

Comment:
Is the language clear enough about why you can no longer access the meeting? Why "ejected" instead of "kicked"? Or even something like, "A moderator has kicked you from the meeting".